AutomatticTracks/src/main/java/com/automattic/android/tracks/TracksClient.java

```diff
@@ -23,13 +24,16 @@
 import java.util.LinkedList;
 import java.util.List;
 
+
+
 public class TracksClient {
     public static final String LOGTAG = "NosaraClient";
 
     public static final String LIB_VERSION = BuildConfig.VERSION_NAME;
     protected static final String DEFAULT_USER_AGENT = "Nosara Client for Android" + "/" + LIB_VERSION;
     protected static final String NOSARA_REST_API_ENDPOINT_URL_V1_1 = "https://public-api.wordpress.com/rest/v1.1/";
-    protected static final int DEFAULT_EVENTS_QUEUE_THREESHOLD = 10;
+    protected static final int DEFAULT_EVENTS_QUEUE_THREESHOLD = 9;
```

Comment:
Hehe, it seems that I'm testing the water, but we want to send events in batch of 10 (and not 11), so the change.